crates/iota-framework/.prettierrc

```
@@ -0,0 +1,7 @@
+{
+    "printWidth": 100,
+    "wrapComments": false,
+    "useModuleLabel": true,
+    "autoGroupImports": "module",
+    "tabWidth": 4
+}
```

```suggestion
{
    "printWidth": 100,
    "useModuleLabel": true,
    "tabWidth": 4
}
```

I feel the defaults are better for these two

---

As discussed on slack, it was decided to keep the defaults.